blog/blog-main.html

```diff
@@ -256,11 +256,10 @@
     <section class="blog-card-post">
       <div class="banner">
         <img
-          src="https://wallpaperaccess.com/full/137490.jpg"
+          src="../images/blog-main-background-image.jpg"
```

Comment:
image looking good. Adjustments need to do
- put the Elzian agro logo in the correct place.
-  According to the image adjust the text in the banner